package.json

```diff
@@ -9,8 +9,12 @@
     "integration-test": "nyc --require ts-node/register mocha --timeout 999999 test/integration/**/*.spec.ts",
     "system-test": "mocha -r ts-node/register --timeout 999999 test/system/**/*.spec.ts",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
-    "start": "node dist/start.js"
+    "start": "node dist/start.js",
+    "dad-gen": "node -r ts-node/register ./source/util/pre-commit.ts && git add ./docs/DataAccess"
```

Comment:
Would `gen-docs` as a task name here be more easily understood by someone not on the team?